autoload/importjs.vim

```diff
@@ -248,9 +248,15 @@ function! importjs#Init()
 
   " vim
   if exists("*job_start")
-    let s:job=job_start(['importjs', 'start', '--parent-pid', getpid()], {
-          \'exit_cb': 'importjs#JobExit',
-          \})
+    if has('win32') || has('win64')
+      let s:job=job_start('importjs.cmd start --parent-pid ' . getpid(), {
```

Comment:
Besides the importjs.cmd chang, it looks like you modified this to also be a single string instead of an array of separate args to job_start. Is that change also important? If not, it would be great to minimize the code duplication here.